DevelopingSpace/starchart
# Refactor dns flows and workers

Close #315 
Close #333 

- Split DNS module into two phases (DB and DNS)
- Specify type to each flow, worker and job
- Change name to subdomain or fqdn to distinguish 

### How to test
Currently integration is only set up in `/dev` route. You don't need any environment setup to test. 

I'm not sure how DNS handling should be initiated after DB update (same frontend call or background job). 
@dadolhay This update requires DB table ID in `addDnsRequest` call in your function. Please let me know if you have any concern with this approach.
